recipe/meta.yaml

```
@@ -22,10 +22,10 @@ build:
 
 requirements:
   host:
-    - python >=3.6,<3.12
+    - python >=3.6,<3.13
     - pip
   run:
-    - python >=3.6,<3.12
+    - python >=3.6,<3.13
```

Since Conan 1.62.0 the issues with Python 3.12 seem to be fixed (see [changelog](https://docs.conan.io/1/changelog.html#nov-2023)). Therefore I think we can just remove the upper bound again.
```suggestion
    - python >=3.6
```

---

Thanks!